tests/python/test_ggui.py

```diff
@@ -51,6 +65,30 @@ def verify_image(window, image_name, tolerance=0.1):
         os.remove(actual_name)
 
 
+def verify_image_array(image, image_name, tolerance=0.1):
+    if image.shape[0] % 2 != 0:
```

Comment:
nit: this seems to overlap with `verify_image` a lot, is it possible to avoid duplicating utils functions?